### ISSUE TITLE
standarization of the registry domain config value for catalogs+apps

### ISSUE THREAD

**Issue by piontec** [giantswarm/roadmap#3179]

**Comment by piontec** [comment]
blocked by https://github.com/giantswarm/roadmap/issues/3185

**Comment by nprokopic** [comment]
> some apps (*-cluster-apps) use strict values.yaml schema validation. Until we have a standard about it, there's again a ton of duplication we have to introduce to make them work (we push all the values to all the apps) and still some app's config can break it

Here if you refer to cluster-\<provider\> and default-apps-\<provider\>, see my comment here https://github.com/giantswarm/roadmap/issues/3187#issuecomment-1931888908

TL;DR We should make a clear distinction between configuration of "regular" apps and cluster-\<provider\> (and default-apps-\<provider\>) apps. They are used for different things and "global" configs (changed "globally") have quite different impacts. Universal config for "regular" apps (that deploy controllers/operators) should IMO not be universal for and shared with cluster-\<provider\> (and default-apps-\<provider\>) apps as well.

**Comment by weatherhog** [comment]
waiting for https://github.com/giantswarm/giantswarm/issues/31101

**Comment by weatherhog** [comment]
@piontec could you please check this issue if there is still work that needs to be done ?

**Comment by piontec** [comment]
it's no longer relevant, dead and rotten
